### PR TITLE
Compatiable with Laravel 7 MigrationCreator construct

### PR DESCRIPTION
**Pull request by virutmath**

### CODE REVIEW

**Comment by czhou** [comment]
Is there any plan to merge this pr please? cause this issue breaks helpers on Laravel 7.

**Comment by hanody968** [comment]
Free Nice

**Comment by themightysapien** [comment]
Why is it not merged yet ??

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     ],
     "require": {
         "php": ">=7.0.0",
-        "encore/laravel-admin": "~1.6"
+        "encore/laravel-admin": "~1.6 || dev-hq_ems"
```
Comment (jxlwqq) [inline composer.json]:
```suggestion
        "encore/laravel-admin": "~1.6"
```

**Review by jxlwqq — CHANGES_REQUESTED** [review]

**Review by jxlwqq — APPROVED** [review]

**Comment by jxlwqq** [comment]
thanks.

**Pull request merged**